src/EventRegistry.php

```diff
@@ -61,7 +64,7 @@ public function __construct(EventDispatcherInterface $eventDispatcher)
      */
     public function getAllActiveEvents(): array
     {
-        $activeEvents = [];
+        $activeEvents = array();
```

Comment:
I think using `[]` is fine. We just need to create a `.php_cs` file to tweak the default fabbot config regarding CS.